zmq.hpp

```diff
@@ -538,8 +538,10 @@ class message_t
 
     /** Dump content to string. Ascii chars are readable, the rest is printed as hex.
          *  Probably ridiculously slow.
+         *  If zero is passed to limit_printable_length, the the string, the message
```

Comment:
I am not sure what this should actually be. "the the string" can be left out entirely, but maybe there is something missing.